### PR TITLE
Make the log level and configuration directory configurable in the init ...

### PR DESCRIPTION
**Pull request by nomis52**
...script.

### CODE REVIEW

**Comment by nomis52** [comment]
@RenZ0  can you confirm this works please.

**Comment by RenZ0** [comment]
Sure, I will try tomorrow.

File: tools/ja-rule/ja-rule-controller.cpp [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+  }
+
+  void UpdateDevice(OpenLightingDevice* device) {
+    m_controller_impl.UpdateDevice(device);
+  }
+
+  void SendRDMRequest(const RDMRequest *request,
+                      ola::rdm::RDMCallback *on_complete) {
+    // TODO(simon): implement me!
+    (void) request;
```
Comment (peternewman) [inline tools/ja-rule/ja-rule-controller.cpp]: OLA_UNUSED? Or is this a temporary thing?

File: tools/ja-rule/ja-rule-controller.cpp [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+  ~Controller() {
+  }
+
+  void UpdateDevice(OpenLightingDevice* device) {
+    m_controller_impl.UpdateDevice(device);
+  }
+
+  void SendRDMRequest(const RDMRequest *request,
+                      ola::rdm::RDMCallback *on_complete) {
+    // TODO(simon): implement me!
```
Comment (peternewman) [inline tools/ja-rule/ja-rule-controller.cpp]: Is this still your WIP? Otherwise it should log a warning or error message if it hits this.

**Comment by peternewman** [comment]
Looks good, although seems like some duplication in the Ja-rule stuff. Will those bits get removed from the other app?

File: tools/ja-rule/ja-rule-controller.cpp (hunk 1 above)
Comment (nomis52) [inline tools/ja-rule/ja-rule-controller.cpp]: It's temporary. I still have work to do :)

File: tools/ja-rule/ja-rule-controller.cpp (hunk 2 above)
Comment (nomis52) [inline tools/ja-rule/ja-rule-controller.cpp]: Yup, still working on it.

**Comment by nomis52** [comment]
The bits in ja-rule-controller form the base of the plugin. Once I've tidied it up some more I plan to move it to plugins/usbdmx .

**Pull request merged**